at_commons/test/at_key_regex_test.dart

```diff
@@ -411,4 +411,28 @@ void main() {
       });
     });
   });
+
+  group('A group of test to validate local keys', () {
```

Comment:
Please add tests for local keys when sharedBy is not set

Reply:
Added unit test when sharedBy is not populated